README.md

```diff
@@ -1 +1,161 @@
-# node-modules-path
\ No newline at end of file
+# Paths, Folders and Modules
+
+## Overview
+
+This lesson will cover how to work with paths, use them when importing modules. We'll also show a code organization technique where you can import folders.
+
+## Objectives
+
+1. Describe `__dirname`, `process.cwd()` and `process.env.PWD`
+1. Describe how to import folders (and why)
+
+
+## Absolute Path with __dirname
+
+Consider the code in which we read a file on Mac OS X. The file is in a `data` folder relative to our script:
+
+```js
+var fs = require('fs')
+var customers = fs.readFileSync('data/customers.csv', {encoding: 'utf8'})
+console.log(customers)
+```
+
+Unfortunately, this code will break on Windows. This is due to a different path syntax, i.e., `data\customers.csv`. That's why we use `path.join()` which we discussed in one of the previous lessons. This example is in the file `customers.js`:
+
+```js
+var fs = require('fs')
+var path = require('path')
+var customers = fs.readFileSync(path.join('data','customers.csv'), {encoding: 'utf8'})
+console.log(customers)
+```
+
+However, if we attempt to run the `customers.js` file from a patent folder with `$ node node-modules-path/customers.js`, the code will produce an error:
```

Comment:
not sure I understand what you're saying - when I try to run that command from the root of this repo, i get `Cannot find module '/Users/victoriafriedman/Desktop/RUBY_LABS/learn-verified/node/unit2/node-modules-path/node-modules-path/customers.js'` where should we be running that command?